instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/test/mongo_test.go

```diff
@@ -78,6 +80,25 @@ func TestDBCrudOperation(t *testing.T) {
 				return false
 			}),
 		},
+		{
+			title: "insert",
+			operation: func(ctx context.Context, db *mongo.Database) (interface{}, error) {
+				return db.Collection("test-collection").InsertOne(ctx, bson.D{{Key: "test-item", Value: "test-value"}})
+			},
+			sanitiser: func(command bson.Raw) string {
```

Comment:
How about a use case when the "sanitizer" returns an empty string? Shouldn't we interpret it as "retraction" and not add `db.statement` attribute at all in such a case?

Reply:
That's a good shout, I'll implement that